downstream/modules/platform/proc-controller-run-job-template.adoc

```diff
@@ -0,0 +1,28 @@
+[id="controller-run-job-template"]
+
+= Running a job template
+
+A benefit of {ControllerName} is the push-button deployment of Ansible playbooks. 
+You can configure a template to store all the parameters that you would normally pass to the Ansible playbook on the command line.  
+In addition to the playbooks, the template passes the inventory, credentials, extra variables, and all options and settings that you can specify on the command line.
+
+.Procedure
+
+* From the navigation panel, select menu:Templates[Launch] next to the job template.
+
+image::controller-gs-job-templates-launch.png[Launch template]
+
+The initial job launch generates a status page, which updates automatically using {ControllerName}'s Live Event feature, until the job is complete. 
+
+For more information on the job results, see link:http://docs.ansible.com/automation-controller/4.4/html/userguide/jobs.html#ug-jobs[Jobs] in the Automation Controller User Guide.
+
+.Additional resources
+
+To learn more about these {ControllerName} features or to learn about administration tasks and the controller API, see the following documentation sets:
+
+* link:http://docs.ansible.com/automation-controller/4.4/html/userguide/index.html#ug-star[Automation Controller User Guide]
```

Comment:
```suggestion
* link:http://docs.ansible.com/automation-controller/latest/html/userguide/index.html#ug-star[Automation Controller User Guide]
```

Optional: these links will probably be replaced once the migration to asciidoctor is done, and  4.4 will always be the most recent version on docs.ansible.com